compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -368,7 +368,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
             );
         };
         if let Some(file) = file {
-            err.note(format!("the full type name has been written to '{}'", file.display()));
+            err.note(format!("the full type name has been written to '{}' consider using --verbose to print it on stdout", file.display()));
```

Comment:
Can you change this to "print it to the console"? I don't think rustc even emits errors to stdout, but instead to stderr.

Reply:
Whoops, that is right. Thank you!

Reply:
Done!